### PR TITLE
Fix to highlight non-expandable top level nav items when active

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Fixes https://jira.coreos.com/browse/ODC-1805

### CODE REVIEW

File: frontend/public/style/_overrides.scss [hunk 1]
```diff
@@ -313,6 +313,11 @@ kbd {
           display: block;
         }
       }
+
+      // Top level items with no sub-items, highlight when active
+      > a.pf-c-nav__link.pf-m-current {
```
Comment (christianvogt) [inline frontend/public/style/_overrides.scss]: Can you remove the `a` ?

```suggestion
      > .pf-c-nav__link.pf-m-current {
```

**Comment by christianvogt** [comment]
Functions well.

**Comment by jeff-phillips-18** [comment]
Thanks! Updated to the simplest case.

**Pull request merged**